bench/workgen/workgen.cpp

```diff
@@ -653,6 +653,8 @@ WorkloadRunner::start_tables_drop(WT_CONNECTION *conn)
             WT_DECL_RET;
             // Spin on EBUSY. We do not expect to get stuck.
             while ((ret = session->drop(session, uri.c_str(), "checkpoint_wait=false")) == EBUSY) {
+                if (stopping)
```

Comment:
Line 290, there is something similar that might be worth looking at:
```
        /*
         * Drop the table. Keep retrying on EBUSY failure - it is an expected return when
         * checkpoints are happening.
         */
        while ((ret = session->drop(session, uri, "checkpoint_wait=false")) == EBUSY)
            sleep(1);
```

Reply:
And 491:
```
                    while (status == EBUSY && ++retries < TABLE_MAX_RETRIES);
```

Reply:
And 617:
```
            while (drops < drop_count) {
```

Reply:
Checked the loops and I believe two of them needed to be fixed and I have done that. Otherwise will exit as needed.